.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
+Thank you for a new Pull Request!
+
+Please, make reviewers work easier by checking, if you are not introducing any technical debt: https://github.com/ManageIQ/guides/blob/master/reviewers_guidelines.md
```

Comment:
I think this belongs under `Guidelines`, right?

Reply:
@bdunne You're right. I'll update the PR. Thanks 👍